docs/bytecode.md

```diff
@@ -21,8 +21,10 @@ this format, which we successfully did for StableHLO
 
 Portable artifacts can be created using either the `stablehlo-translate` tool,
 or directly in C++ or Python APIs. Serialization needs a target version of
-StableHLO to write an artifact (the current version is 0.10.0). Deserialization
-uses the current version of StableHLO to read an artifact.
+StableHLO to write an artifact written in `#.#.#` format (See [Version.h](
+https://github.com/openxla/stablehlo/search?q=filename%3AVersion.h+getCurrentVersion)
```

Comment:
When I'm following this link, I'm getting "We couldn’t find any code matching 'filename:Version.h getCurrentVersion' in openxla/stablehlo".

Reply:
I may just link to the file. If you then refresh that page, it will magically work. Same happened to me, github search isn't great.